samples/zigbee/light_bulb/prj.conf

```diff
@@ -43,3 +39,11 @@ CONFIG_NET_IPV6_NBR_CACHE=n
 CONFIG_NET_IPV6_RA_RDNSS=n
 CONFIG_NET_IP_ADDR_CHECK=n
 CONFIG_NET_UDP=n
+
+# BLE stack is required for MPSL compatible flash driver.
+CONFIG_BT=y
+CONFIG_BT_HCI=y
+CONFIG_BT_CTLR=y
+CONFIG_BT_LL_NRFXLIB=y
```

Comment:
This one and the following are undefined symbols. Maybe something has changed with the upmerge?

Reply:
Yes, they are undefined. This PR is not ready for a review. This change is introduced by a simple cherry-pick of a commit from the past.

Reply:
Building fixed!
I tried to go through Kconfig diffs and disable some of the BLE features. I will keep them removed if they have any impact on the flash & RAM usage.